static/src/js/components/Panels/PanelItem.js

```diff
@@ -113,6 +114,43 @@ export class PanelItem extends Component {
       }
     ])
 
+    const renderChildren = () => {
+      const contents = (
+        <>
+          {
+            typeof children === 'string' && children
+          }
+          {
+            typeof children !== 'string' && (
```

Comment:
If not a string, what is this? What is the use case here for a string versus.. anything else?

Reply:
If the child is not a string, it's a react component. In the case it's a string, we just want to render the string. If its a component, we use cloneElement to set some props on the children that get passed in. This is really just a safety net so we don't attempt to set props on a string, which would not work.